lib/toxiproxy.rb

```diff
@@ -54,6 +53,11 @@ def self.all
     Collection.new(proxies)
   end
 
+  # Sets the toxiproxy host to use.
+  def self.host=(host)
+    @uri = host && ::URI.parse(host)
```

Comment:
@Sirupsen did you still want to allow a string, or assume we're always giving it a URI? I updated the code to allow for both strings and URIs.

@pushrax You're right – and, to some extent, I believe that `nil` makes sense (absence of value: do what you'd do if I've never set the host, which to me also means the absence of value), but I opted out of that interpretation :smile: